src/modules/statement/comment_doc.rs

```diff
@@ -47,7 +47,7 @@ impl SyntaxModule<ParserMetadata> for CommentDoc {
                                 continue;
                             }
                             let delimiter = if last_char == '\n' { "" } else { " " };
-                            self.value.push_str(&format!("{}{}", delimiter, token.word[3..].trim()));
+                            self.value.push_str(&format!("{}{}  ", delimiter, token.word[3..].trim()));
```

Comment:
What's the reason of this whitespace in the end?

Reply:
What do you mean? I'm not sure I understood

Reply:
It's intended behaviour. It's so that it looks better in your editor. If you want new line you add empty line, like this:
```
/// hello
///
/// world
```

you can see this in [formatter.rs](https://github.com/amber-lang/amber/blob/master/src/modules/formatter.rs) for example

Reply:
Usually this isn't the normal behavior in comments, in js/php/python and many others it is normal to have multiple lines and the documentation add a newline automatically.

Reply:
It's the same as vscode displays it for rust on hover tooltip

Reply:
Okay I agree with @KrosFire on this. This is something that makes sense in Markdown. In Markdown you can insert break  line like this:
```md
This is on 
the same line

This is a
different line
```